fourkitchens/sous-drupal-project
# remove profile: set gin as administration theme

**Description:**

The version of sous with the profile set Gin as the default theme. With the new version, it's currently not setting any admin theme which makes it difficult to navigate the admin. 
- enable the Gin theme
- set the Gin theme as the admin theme
- set node edit to use the admin theme
- add emulsify_twig to composer

**Testing steps**
Install the site from config. Verify the Gin theme is set as the admin theme and works as expected.
